examples/hello_world/module/rust/src/lib.rs

```diff
@@ -43,6 +43,12 @@ impl oak::Node for Node {
 
 impl HelloWorldNode for Node {
     fn say_hello(&mut self, req: HelloRequest) -> GrpcResult<HelloResponse> {
+        if req.greeting == "Query-of-Error" {
```

Comment:
Alternatively, create yet another method `say_hello_error` that always does that?

Reply:
I left this as-is to avoid proliferation of gRPC methods.